### PR TITLE
LPS-164147 [EVP] Create the Request Object 

### PR DESCRIPTION
**Pull request by ytuik**

### CODE REVIEW

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

**Review by silasbsi — APPROVED** [review]
LGTM :martial_arts_uniform: :+1:

**Comment by silasbsi** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/list-type-definitions/grant-request-type.list-type-entries.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[
+	{
+		"key": "blank",
+		"name": " ",
+		"name_i18n": {
+			"en-US": " "
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/list-type-definitions/grant-request-type.list-type-entries.json:6]: Please substitute the blanck spaces for some content, this breaks the site building haha
```suggestion
		"name": "No Grant",
		"name_i18n": {
			"en-US": "No Grant"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/list-type-definitions/request-status.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+	"name": "RequestStatus",
+	"name_i18n": {
+		"en-US": "Request Status"
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/list-type-definitions/request-status.json:4]: Following your naming pattern it should be: 
```suggestion
		"en-US": "Request: Request Status"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/list-type-definitions/request-behalf.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+	"name": "GroupBehalf",
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/list-type-definitions/request-behalf.json:2]: Substitute Group for Request
```suggestion
	"name": "RequestBehalf",
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/organization.json [hunk 4]
```diff
@@ -8,36 +8,38 @@
...
-			"name": "createDate",
+			"name": "city",
 			"objectFieldSettings": [
+				{
+					"name": "showCounter",
+					"value": "false"
+				}
 			],
-			"required": false,
+			"required": true,
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/organization.json:39]: Could you change the required fields in this file to _false_?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/request.json [hunk 5]
```diff
@@ -79,33 +79,68 @@
...
 			"DBType": "Date",
 			"businessType": "Date",
 			"defaultValue": "",
-			"indexed": true,
+			"indexed": false,
 			"indexedAsKeyword": false,
 			"indexedLanguageId": "",
 			"label": {
-				"en_US": "Date"
+				"en_US": "Start Date of Service"
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/request.json:141]: Change it to 
```suggestion
				"en_US": "Service Start Date"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/request.json [hunk 6]
```diff
@@ -114,40 +149,53 @@
...
+			"DBType": "Date",
+			"businessType": "Date",
 			"defaultValue": "",
-			"indexed": true,
+			"indexed": false,
 			"indexedAsKeyword": false,
 			"indexedLanguageId": "",
 			"label": {
-				"en_US": "File Entry Id"
+				"en_US": "End Date of Service"
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/request.json:159]: Change it to
```suggestion
				"en_US": "Service End Date"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/request.json [hunk 7]
```diff
@@ -157,50 +205,45 @@
...
 			"indexedAsKeyword": false,
-			"indexedLanguageId": "",
 			"label": {
-				"en_US": "Status"
+				"en_US": "Request Status"
 			},
-			"name": "status",
-			"objectFieldSettings": [
-			],
+			"listTypeDefinitionId": "[$LIST_TYPE_DEFINITION_ID:Request Status$]",
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/request.json:244]: Change it to:
```suggestion
			"listTypeDefinitionId": "[$LIST_TYPE_DEFINITION_ID:Request: Request Status$]",
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-relationships/request-organization-entry-to-request.json [hunk 8]
```diff
@@ -0,0 +1,11 @@
+{
+	"deletionType": "prevent",
+	"label": {
+		"en_US": "Organization ID"
+	},
+	"name": "evpOrganizationId",
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-relationships/request-organization-entry-to-request.json:6]: Take out the "ID":
```suggestion
		"en_US": "Organization"
	},
	"name": "organization",
```

Comment (JoMaAlves) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-relationships/request-organization-entry-to-request.json:6]: Don't know why, but the curly brace and comma ("},") got red. Do not erase it, maintain it the way it is.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-relationships/request-organization-entry-to-request.json [hunk 9]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-relationships/request-organization-entry-to-request.json:1]: Rename the file to "organization-entry-to-request"

**Review by JoMaAlves — CHANGES_REQUESTED** [review]
🛠️ Needs some changes 🛠️

**Review by juliocezarjc — APPROVED** [review]
Nice job! 🤙

**Comment by juliocezarjc** [comment]
ci:test:relevant

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

**Comment by juliocezarjc** [comment]
ci:test:relevant

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

**Comment by juliocezarjc** [comment]
ci:test:sf

**Review by JoMaAlves — APPROVED** [review]
LGTM ​🦖​🌋​ 
Thank you @ytuik

**Comment by JoMaAlves** [comment]
ci:test:relevant

**Comment by juliocezarjc** [comment]
Just starting reviewing :)

**Comment by silasbsi** [comment]
ci:forward

**Comment by silasbsi** [comment]
Just starting reviewing :)

**Pull request closed without merging**